config/clouddriver.yml

```diff
@@ -51,6 +51,13 @@ google:
       jsonPath: ${providers.google.primaryCredentials.jsonPath}
       consul:
         enabled: ${providers.google.primaryCredentials.consul.enabled:false}
+      # Specify the project id of each GCP project whose images you want indexed and
```

Comment:
Good suggestion. Have done so here: http://www.spinnaker.io/v1.0/docs/target-deployment-configuration#section-google-cloud-platform

I'm assuming by 'built-in defaults' you're referring to the public cloud projects. Those are configured here: https://github.com/spinnaker/clouddriver/blob/master/clouddriver-web/config/clouddriver.yml#L137-L145, are available to _all_ Google accounts, and are not affected by the `imageProjects` attribute. Note that the `imageProjects` attribute has been in place for a long time and we're just adding some documentation in response to a user query.
> The "should look similar to" isnt clear. Do you mean it is usually in the form?
Yes. I did several experiments and each account I observed that worked properly had that form.